main.cpp

```diff
@@ -0,0 +1,15 @@
+
+#include "laba5.h"
```

Comment:
you changed the name of the header but did not change in the include, rename please.